### ISSUE TITLE
Tablet features

### ISSUE THREAD

**Issue by Tom1235** [sumatrapdfreader/sumatrapdf#110]
I use this on my surface pro 3 and i love it but some tablet feature would be nice: tap(click) on the right side -> next page, tap on left side -> previous page, long press or double tap to exit full screen mode, full screen on button. And an option to disable scrollbars in single page view. Thank you for your work.
Thomas

**Comment by R74** [comment]
joining Tom1235. It would be great to add tablet support int this great software. 
1) pages layout automatically reflecting display orientation (1 page for portrait, 2 pages for landscape)
2) double-click/longer tap the display leads to exit fullscreen or to menu with several commands (exit app, exit fullscreen, change page layout - in case point 1 is not possible automatically)
3) possibility to open lastly opened file on sumatra start

**Comment by GitHubRulesOK** [comment]
Multiple issues make tracking difficult
tom1235
1) tap each side is duplicated by https://github.com/sumatrapdfreader/sumatrapdf/issues/1203
2) long hold to exit full screen is now posible thus closed
3) was covered later in https://github.com/sumatrapdfreader/sumatrapdf/issues/191

R74
1) Whilst it may be useful to some, automatic changes due to differing hardware such as accelerometers are not generally handled by SumatraPDF code. Raise a single issue if you think this is reasonably achievable. (examples of similar common C++  code functionality always helps)
2) Double tap and alternative long hold exit option should now both be functional
3) Has been covered in https://github.com/sumatrapdfreader/sumatrapdf/issues/1055#

In summary all the above are closed or to be addressed elsewhere so am closing this thread
